app/assets/scripts/components/map/node-layer.js

```diff
@@ -27,106 +25,89 @@ export default function NodeLayer({
-      id: `${activeParameter}-squares`,
+      id: `${activeParameter}-nodes`,
       source: sourceId,
       'source-layer': 'default',
       type: 'symbol',
       paint: {
         'icon-color': defaultColor,
       },
       layout: {
-        'icon-image': 'square',
-        'icon-size': coloredSquareSize,
+        'icon-image': iconMatch,
+        'icon-size': coloredSymbolSize,
         'icon-allow-overlap': true,
       },
-      filter: squaresFilter,
-    });
-
-    map.addLayer({
-      id: `${activeParameter}-circle-outline`,
-      source: sourceId,
-      'source-layer': 'default',
-      type: 'circle',
-      paint: {
-        'circle-color': defaultColor,
-        'circle-opacity': 1,
-        'circle-radius': borderCircleRadius,
-        'circle-blur': 0,
-      },
-      filter: circlesFilter,
-    });
-
-    map.addLayer({
-      id: `${activeParameter}-circles`,
-      source: sourceId,
-      'source-layer': 'default',
-      type: 'circle',
-      paint: {
-        'circle-color': defaultColor,
-        'circle-opacity': circleOpacity,
-        'circle-radius': coloredCircleRadius,
-        'circle-blur': circleBlur,
-      },
-      filter: circlesFilter,
     });
 
     // Change the cursor to a pointer when the mouse is over the layer.
-    map.on('mouseenter', `${activeParameter}-circles`, function () {
-      map.getCanvas().style.cursor = 'pointer';
-    });
-    map.on('mouseenter', `${activeParameter}-squares`, function () {
+    map.on('mouseenter', `${activeParameter}-nodes`, function () {
       map.getCanvas().style.cursor = 'pointer';
     });
 
     // Change it back to a pointer when it leaves.
-    map.on('mouseleave', `${activeParameter}-circles`, function () {
-      map.getCanvas().style.cursor = '';
-    });
-    map.on('mouseleave', `${activeParameter}-squares`, function () {
+    map.on('mouseleave', `${activeParameter}-nodes`, function () {
       map.getCanvas().style.cursor = '';
     });
 
     return () => {
-      if (map.getLayer(`${activeParameter}-squares`))
-        map.removeLayer(`${activeParameter}-squares`);
-      if (map.getLayer(`${activeParameter}-square-outline`))
-        map.removeLayer(`${activeParameter}-square-outline`);
-      if (map.getLayer(`${activeParameter}-circles`))
-        map.removeLayer(`${activeParameter}-circles`);
-      if (map.getLayer(`${activeParameter}-circle-outline`))
-        map.removeLayer(`${activeParameter}-circle-outline`);
+      if (map.getLayer(`${activeParameter}-nodes`))
+        map.removeLayer(`${activeParameter}-nodes`);
+      if (map.getLayer(`${activeParameter}-node-outline`))
+        map.removeLayer(`${activeParameter}-node-outline`);
     };
   }, [activeParameter]);
 
+  useEffect(() => {
+    map.setPaintProperty(`${activeParameter}-node-outline`, 'icon-color', [
+      'case',
+      [
+        'in',
+        ['number', ['get', 'locationId']],
+        ['literal', allSelectedLocations],
+      ],
+      unusedBorderColor,
```

Comment:
I would recommend a different color, I think this is a little hard to recognize against the blue